.github/workflows/ci.yml

```diff
@@ -196,12 +199,17 @@ jobs:
         with:
           string: ${{ github.repository }}
 
-      - name: Deploy images
+      - name: Download stored image from artifacts
+        uses: actions/download-artifact@v3.0.0
         env:
-          REPO_NAME: ${{ steps.github-repository-name-case-adjusted.outputs.lowercase }}
-          SHA: ${{ github.sha }}
+          VAPP_IMAGE: ${{ matrix.component.Name }}_${{ github.sha }}-amd64.tar
```

Comment:
Is it intended to refer to a certain cpu arch (amd64) here?

Reply:
Yes, for some reason (history: could be execution time) the integration test runs only for one architecture.

Reply:
I think I still need to remove REPO_NAME and SHA because they are not used anymore :)